README.md

```
@@ -12,9 +12,9 @@
   <br/>
   <p>
   
-[![GitHub license](https://badgen.net/github/license/wes4m/zatca-xml-js?v=0.1.0)](https://github.com/wes4m/zatca-xml-js/blob/main/LICENSE)
-    <a href="https://github.com/wes4m">
-        <img src="https://img.shields.io/badge/maintainer-wes4m-blue"/>
+[![GitHub license](https://badgen.net/github/license/Repzo/zatca-xml-js?v=0.1.0)](https://github.com/Repzo/zatca-xml-js/blob/main/LICENSE)
+    <a href="https://github.com/Repzo">
+        <img src="https://img.shields.io/badge/maintainer-Repzo-blue"/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)